chapters/managementSummary.typ

```diff
@@ -32,62 +60,34 @@ Replaces the defined function parameter type with the type `auto` and removes th
     ]
   ),
   caption: [ Example of "Abbreviate Function Template" refactoring ],
-)
-]
-
-/ Inline Concept Requirement : #[
-Inlines the defined requirements into the template definition.
-#figure(
-  table(
-    columns: (1fr, 1fr),
-    align: horizon,
-    [*Before*], [*After*],
-    ```cpp
-    template <typename T>
-    void foo(T) requires std::integral<T> {}
-    ```,
-    ```cpp
-    template <std::integral T>
-    void foo() {}
-    ``` ,
-  ),
-  caption: [ Example of "Inline Concept Requirement" refactoring ],
-)
-]
-
-The implemented refactoring features are added to the clangd language server in the `refactor/tweaks` directory.
-@refactoring_contribution shows VSCode using the clangd language server to display refactorings in the code editor.
-In the example in @refactoring_contribution VSCode is communicating with the language server using LSP. 
-To use clangd in VSCode the extension "clangd" can be used.
+) <management_summary_abbreviate>
 
 #figure(
-  image("../drawio/refactoring_contribution.drawio.png"),
+  image("../drawio/refactoring_contribution.drawio.png", width: 90%),
   caption: [
     Diagram showing integration of implemented refactoring
   ],
 ) <refactoring_contribution>
 
 #pagebreak()
+#set heading(numbering: none)
 
-/ Key findings : #[
-- Parts of the code within the LLVM Project are quite old and use older language features. 
-- Pull requests often take a significant amount of time for reviewers to approve the changes.
+=== Key Findings
```

Comment:
Bin ned so happy mit dene key findings, aber mer chemet au ned wirklich besseri in Sinn. Meinsch mer chends eifach weg lah?

Reply:
hmm ich hett ez mal ufs review vode anderne gwartet bevor mers usenehmed finds eig na guet sochli Pünkt z highlighte